tests/test_client.py

```diff
@@ -273,3 +274,23 @@ def test_client_request_timed_out(request_mock, client):
     with pytest.raises(RequestError) as err:
         client.payments.list()
     assert "Read timed out." in str(err.value)
+
+
+def test_client_data_consistency_error(client, response):
+    """When the API sends us data we did not expect raise an consistency error."""
+    order_id = 'ord_kEn1PlbGa'
+    line_id = 'odl_12345'
+    response.get('https://api.mollie.com/v2/orders/{order_id}'.format(order_id=order_id), 'order_single')
+    response.patch('https://api.mollie.com/v2/orders/{order_id}/lines/{order_line_id}'.format(
+        order_id=order_id, order_line_id=line_id), 'order_single')
+
+    order = client.orders.get(order_id)
+    data = {
+        "name": "LEGO 71043 Hogwarts™ Castle",
+    }
+    # Update an nonexistent order line. This raises an data consistency error.
+    with pytest.raises(DataConsistencyError) as err:
+        order.update_line(line_id, data)
+    exception = err.value
```

Comment:
Please use `pytest.raises(DataConsistencyError, match=)` to check the error message